app/models/commissioning_document.rb

```diff
@@ -33,12 +26,13 @@ class CommissioningDocument < ApplicationRecord
     probation: "probation",
     security: "security",
     telephone: "telephone",
+    standard: "standard",
   }
 
   belongs_to :data_request_area
   belongs_to :attachment, class_name: "CaseAttachment"
 
-  validates :data_request, presence: true
+  validates :data_request_area, presence: true
```

Comment:
How are you going to deal with existing cases?

Reply:
Ive added `if: :has_no_request_area?` 

without this it would break the commissioning_document download (not too sure why) the document would download as "download.1".